test/built-ins/Promise/prototype/finally/rejected-observable-then-calls-argument.js

```diff
@@ -19,7 +19,7 @@ flags: [async]
 
 Promise.reject(new Test262Error())
   .finally(function() {})
-  .then($DONE, $DONE);
+  .then(() => $DONE(), $DONE);
```

Comment:
Nice! I didn't get a chance to do this one (which would've taken me much longer since I need to first understand this new feature). Thanks for addressing it quickly!